src/components/PostDetails/PostDetails.tsx

```diff
@@ -1,45 +1,81 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { NewCommentForm } from '../NewCommentForm';
 import './PostDetails.scss';
+import { getPostDetails } from '../../api/post';
+import { CommentsList } from '../CommentsList';
+import { ForComment } from '../../Types/Comment';
+import { createCommemt, deleteComment } from '../../api/comment';
 
-export const PostDetails: React.FC = () => (
-  <div className="PostDetails">
-    <h2>Post details:</h2>
-
-    <section className="PostDetails__post">
-      <p>sunt aut facere repellat provident occaecati excepturi optio</p>
-    </section>
-
-    <section className="PostDetails__comments">
-      <button type="button" className="button">Hide 2 comments</button>
-
-      <ul className="PostDetails__list">
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>My first comment</p>
-        </li>
-
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>sad sds dfsadf asdf asdf</p>
-        </li>
-      </ul>
-    </section>
-
-    <section>
-      <div className="PostDetails__form-wrapper">
-        <NewCommentForm />
-      </div>
-    </section>
-  </div>
-);
+type Props = {
+  postId: number,
+};
+
+export const PostDetails: React.FC<Props> = ({ postId }) => {
+  const [comments, getComments] = useState<ForComment[]>([]);
```

Comment:
```suggestion
  const [comments, setComments] = useState<ForComment[]>([]);
```
Эта функция устанавливает новое значение для комментариев, а не возвращает его